js/securesubmit/checkout-form.js

```diff
@@ -339,9 +339,8 @@ document.observe('dom:loaded', function () {
             pEle.insertBefore(ele, pEle.childNodes[2]);
         } catch (e) {
         }
-        try {
-            document.querySelector('#payment-buttons-container').remove();
-        } catch (e) {
+        if(document.getElementById('payment-buttons-container')) {
+            document.getElementById('payment-buttons-container').remove();
```

Comment:
@ttnnkkrr These calls to `document.getElementById` should be abstracted to a single call. This removes duplication as well as the multiple queries against the DOM. Also, I just remembered that older versions of IE have issue with `node.remove()`, so nodes should be removed by calling `node.parentNode.removeChild(node)` which is supported. Both are written below as an example:

```javascript
var container = document.getElementById('payment-buttons-container');
if (container && container.parentNode) {
    // container.parentNode should always exist, but we're playing it safe above
    container.parentNode.removeChild(container);
}
```